frontend/src/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,56 @@
+import './ProfilePage.scss';
+import React, {useContext, useEffect, useState} from "react";
+import BagPlace from "../models/BagPlace";
+import {getBagPlacesOfUser} from "../service/RequestService";
+import BagPlaceCounter from "../components/BagPlaceCounter";
+import {AuthContext} from "../context/AuthProvider";
+import {Button} from "@mui/material";
+import {useNavigate} from "react-router-dom";
+
+export default function ProfilePage() {
+
+    const [bagPlaces, setBagPlaces] = useState<BagPlace[]>([])
+
+    const {token} = useContext(AuthContext)
+
+    const navigate = useNavigate()
+
+    const setupBagPlaces = () => getBagPlacesOfUser(token).then(data => setBagPlaces(data))
+
+    useEffect(() => {
+        setupBagPlaces().catch(e => console.log(e.message))
+
+        //eslint-disable-next-line
+    }, [])
+
+    const handleSettingsButton = () => {
+        navigate('/settings')
+    }
+
+    console.log(bagPlaces)
+
+    return (
+        <div className="profilePage">
+            <div className="upperBox">
+                    <img className="profilePicture"
+                        src={"https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460__340.png"}
+                        alt=""/>
+                <div className="userInfo">
+                    <h3>{"username"}</h3>
```

Comment:
unnecessary code escaping

Reply:
The "username" is also only an example to visualize the design